spec/requests/rails_spec.rb

```diff
@@ -42,6 +42,11 @@
       end
     end
 
+    it 'can return an empty array' do
+      get '/tables', params: { filter: { "name" => "Never Match" } }, headers: { authorization: 'k0kubun' }
+      expect(response.status).to eq(200)
+    end
```

Comment:
Looks like this test case passes on the current `master` branch.
I am afraid that the changes are not covered.

Reply:
I think it would be clearer if the behavior change is covered in the fixture files (JSON/YAML).
```
          items:
            type: object
            properties:
              id:
                type: integer
              ...
            nullable: true

===>


          items:
            type: object
            properties:
```